components/navigation-bar.tsx

```diff
@@ -1,35 +1,26 @@
-const navigationItems = [
-  {
-    url: "https://cesko.digital/projects",
-    name: "Projekty",
-  },
-  {
-    name: "Blog",
-    url: "/",
-  },
-  {
-    url: "https://www.darujme.cz/projekt/1203553",
-    name: "Přispět",
-  },
-];
+import {Router, withRouter} from 'next/router';
+import Link from 'next/link'
 
-const NavigationBar: React.FC = () => (
+interface Props {
+  router: Router;
+}
+
+const NavigationBar: React.FC<Props> = (props) => (
   <div className="header-wrapper">
     <div className="navigation-bar">
       <div>
         <a href={"https://cesko.digital"} className="logo" />
       </div>
       <div>
         <div className="toolbar-links">
-          {navigationItems.map((item, index) => (
-            <a className="toolbar-link" key={index} href={item.url}>
-              {item.name}
-            </a>
-          ))}
+          {props.router.asPath !== '/' &&
+          <Link href="/">
```

Comment:
Tohle dělá client-side navigaci, chceme to? Já zatím asi nevidím důvod proti, jen že všude (na webu i blogu) zatím používáme regulérní reload v prohlížeči.

Reply:
Na blogu bysme vlastně tu client-side navigaci mohli vyzkoušet kompletně, tedy i opačným směrem, z titulky na článek.

Reply:
Já jsem v tom n00b, ale linter se na mně zlobil, když jsem tam měl normální `<a href="">`...

Reply:
Aha! On je ten linter trochu mimo mísu, protože ty dvě věci dělají něco výrazně jiného. Jak tomu rozumím já, `next/link` dělá navigaci čistě na klientovi, tj. bez regulérního round-tripu na server, zatímco `a` je klasický „hloupý“ odkaz. Zkusil jsem nahodit ve #260, jak by to vypadalo, kdybychom přes `next/link` načítali i články. Zatím se mně to moc líbí, je to hodně rychlé.